src/app/splash_models.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+
+from pydantic import BaseModel, HttpUrl
+
+
+class Header(BaseModel):
+    """A single key value header pair."""
+
+    name: str
+    value: str
+
+
+class Cookie(BaseModel):
+    """See http://www.softwareishard.com/blog/har-12-spec/#cookies"""
+
+    name: str
+    value: str
+    httpOnly: bool
+    secure: bool
+    # currently not used and hence not required
```

Comment:
Then why do you have this commented? I would remove it, if it is not necessary

Reply:
will do!

Reply:
done